ecosystem/ffmpeg_plugin/mtl_common.c

```diff
@@ -53,8 +53,7 @@ mtl_handle mtl_dev_get(AVFormatContext* ctx, const struct StDevArgs* args, int*
         return NULL;
       }
     }
-    p.tx_queues_cnt[i] = args->tx_queues_cnt[i] ? args->tx_queues_cnt[i] : 8;
```

Comment:
Are you sure that default from MTL_RX_DEV_ARGS and MTL_TX_DEV_ARGS? Gstreamer ignores the defaults.
Did you checked that?